SAML/example_metadata.xml

```diff
@@ -0,0 +1,9 @@
+<md:EntityDescriptor xmlns:md="urn:oasis:names:tc:SAML:2.0:metadata" entityID="https://client_name.app.salemove.com">
```

Comment:
I think two examples would be good, one for .com and one for .eu

Reply:
Do we still use "salemove.com" in this context or is it possible to use "glia.com" as well?

Reply:
Currently only salemove.com is used: https://salemove.atlassian.net/browse/TM-10809

Reply:
Ok, good!